app/controllers/providers/own_homes_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Providers
+  class OwnHomesController < ApplicationController
+    include ApplicationDependable
+    include Steppable
+    def show
+      @form = LegalAidApplications::OwnHomeForm.new(current_params)
+    end
+
+    def update
+      @form = LegalAidApplications::OwnHomeForm.new(form_params)
+
+      if @form.save
+        if @form.own_home_no?
+          render plain: 'Holding page: 2a. Does your client have any savings or investments?'
+        else
+          render plain: 'Holding page: 1b. How much is your client’s home worth'
+        end
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def current_params
+      legal_aid_application.attributes.symbolize_keys.slice(:own_home)
```

Comment:
Didn't you make some modification to avoid this but instead pass the model to the form?

Reply:
I'm an idiot.